cpp/ql/lib/change-notes/2022-08-17-deleted-deprecations.md

```diff
@@ -0,0 +1,6 @@
+---
+category: minorAnalysis
+---
+* All deprecated predicates/classes/modules that have been deprecated for over a year have been 
+deleted.
```

Comment:
Is this true?  A quick search of the CPP code base suggests there are a number of exceptions where we have not deleted older deprecated predicates, e.g. `File.getShortName()`???

Reply:
According to `git blame` the line containing the `deprecated` annotation for `File.getShortName()` was last touched 12 months ago.  (In [this commit](https://github.com/github/codeql/commit/6c32b92929473d4b020c18c07ebe60d3c6d0090c)). 
That is less than the 14 months required by my script, so it was therefore not deleted.  

That predicate has been deprecated for 4 years, but my script just looks at when the line containing the `deprecated` annotation was last touched.  

I could change the change-note to say "Some" or "Most"?

Reply:
Sounds good.  It does look like `File.getShortName()` might just be an unusual case that got my attention.